wordle-web/components/WordList.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-bottom-sheet
+    v-model="modelValue"
+    max-height="600"
+    class="pa-2 d-flex flex-wrap justify-center align-center"
+  >
+    <v-card>
+      <v-autocomplete
+        v-model="selectedWord"
+        prepend-inner-icon="mdi-magnify"
+        :items="validWords()"
+        placeholder="Enter a Word!"
+        @update:modelValue="updatePage"
+      >
+      </v-autocomplete>
+      <v-btn
+        height="40px"
+        class="align-center d-flex justify-center"
+        v-for="word in pagedWords"
+        :key="word"
+        @click="addGuess(word)"
+      >
+        {{ word.toUpperCase() }}
+      </v-btn>
+      <v-card-item class="ma-3">
+        <v-pagination
+          density="compact"
+          v-model="currentPage"
+          :length="totalPages"
+        />
+      </v-card-item>
+    </v-card>
+  </v-bottom-sheet>
+  <v-btn
+    position="fixed"
+    location="bottom right"
+    class="mr-3 mb-3"
+    size="default"
+    color="primary"
+    elevation="8"
+    rounded
+    @click="modelValue = true"
+    >Word List: {{ validWords().length }}</v-btn
```

Comment:
Depending on the interpretation of the wording of the requirements, we might have to update on every registered keystroke to narrow down available words.

Reply:
I am not totally sure on this one, the way they described it in class makes me think we narrow down the word list after the guess is submitted.